sql/lessons_summary.txt

```diff
@@ -0,0 +1,128 @@
+SQL student path
+
+https://markdown-here.com/livedemo.html
+https://github.com/adam-p/markdown-here/wiki/Markdown-Here-Cheatsheet
+
+List of #fiddle
+#fiddle-5bd3754272a88o2ajnqge8i9
+#fiddle-5be8825d749e251ljod9s39t
+#fiddle-5be888fa749e451ljodase3y
+
+Lesson 1:
+- Explanation of the lessons. "By the end of this lessons you'll be able to: ..."
+- What is SQL?
+- What can it do?
+- What is a Database?
+- Short explanation about https://sqliteonline.com/
+- Columns and rows
+- SELECT (including columns and row filtering): WHERE statement: =, <>, <, >
+- ANNEX: Data set in SQL command lines in case they want to load it in other DB.
+- SELECT exercises: Report requirements so they build the query. The expected output can be provided so they can validate the correctness of the query.
+    - Show all the data available of all the persons.
+    - Show the name and last name of all the persons 20 years old or above.
+    - Show the name and last name of all the persons living in UK.
+    - Show the name and last name of all the persons not living in their birth place.
+    - Show the distinct studies of all the persons.
+    - Show the distinct countries of residence of all the persons.
+    - Show the locations cities in UK.
+
+
+Lesson 2:
+- Operators: IN, NOT IN.
+- ORDER BY, LIMIT
+- NULL value, IS and IS NOT
+- SELECT exercises:
+    - Show the name and last name of all the persons. Sort the result by last name in ascending order, in case of equal last name sort by first name.
+    - Show the last name and the biological sex of the person with higher number of children.
+    - Show the country of birth of the oldest person.
+    - Show the age of the oldest person.
+    - Show all the host names.
+    - Show the host name and the student capacity. Sort the result in descending order.
+    - Pick a host ID and show the dates where workshops were held there. Order the result chronologically.
+    - Show the top 5 host according to their capacity
+    - Show the distinct ID of the persons that RSVP but then they did not show up (attendance = false).
+
+
+
+Lesson 3:
+- Functions in WHERE clause: MAX, MIN, COUNT, AVG
+- Functions in SELECT clause: UPPER,
+- GROUP BY
+- HAVING
```

Comment:
This is not in tutorial 3. It's also mentioned in tutorial 7, but I only see tutorials 1-6. Is this page up to date?

Reply:
I will remove this file, it's not updated and should not be in the PR either.